modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -70,21 +73,18 @@ NAME                            STATUS
 ∙ image-digest-exporter-xlkn7   Completed
 ----
 
-. Retrieve the signature and payload from the object stored as `base64` encoded annotations:
+. Retrieve the signature from the object stored as `base64` encoded annotations:
 +
 [source,terminal]
 ----
 $ export TASKRUN_UID=$(tkn tr describe --last -o  jsonpath='{.metadata.uid}')
-$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" > signature
-$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/payload-taskrun-$TASKRUN_UID}" | base64 -d > payload
+$ tkn tr describe --last -o jsonpath="{.metadata.annotations.chains\.tekton\.dev/signature-taskrun-$TASKRUN_UID}" | base64 -d > sig
 ----
 
-. Verify the signature.
+. Verify the signature using the public key that you created. Replace `path/to/cosign.pub` with the path name of the public key file.
 +
 [source,terminal]
 ----
-$ cosign verify-blob --key k8s://openshift-pipelines/signing-secrets --signature ./signature ./payload
+$ cosign verify-blob-attestation --insecure-ignore-tlog --key path/to/cosign.pub --signature sig --type slsaprovenance --check-claims=false /dev/null
 Verified OK
```

Comment:
per above guidelines, code blocks and output must be separated:
![image](https://github.com/openshift/openshift-docs/assets/40172997/ba239f96-db68-4478-8d69-f59c9a240f9d)